### ISSUE TITLE
normalize_image data augmentation causing severe drop in mAP

### ISSUE THREAD

**Issue by tetchart3** [tensorflow/models#3185]
### System information
- **What is the top-level directory of the model you are using**: object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.4.0
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: CUDA 8, cuDNN 6
- **GPU model and memory**: 1080Ti 11 GB
- **Exact command to reproduce**:

### Describe the problem
I am trying to use the "faster_rcnn_resnet101" object detection framework. With no data augmentations, I can achieve about .6 mAP on my problem. My images are in the 0 to 255 range so to use the data augmentations that require the images to be in the 0 to 1 range I use the normalize_image function to move the image to the correct range and then I do it again after all the augmentations to make sure they are back to the 0 to 255 range. Since trying this technique I've seen my mAP drop to basically 0 (the last run I got a .0078). I tried doing no data augmentations between shifting my images from 0 to 255 to 0 to 1 and then back again, but that still causes my mAP to drop to 0. This prevents me from using any data augmentation option that requires the images to be in the 0 to 1 range.

I would expect this config would produce the same results as a config that had no data augmentation options as the two data augmentations should just cancel out.

```
model {
  faster_rcnn {
    num_classes: 93
    image_resizer {
      keep_aspect_ratio_resizer {
        min_dimension: 812
        max_dimension: 812
      }
    }
    feature_extractor {
      type: "faster_rcnn_resnet101"
      first_stage_features_stride: 16
    }
    first_stage_anchor_generator {
      grid_anchor_generator {
        height_stride: 4
        width_stride: 4
        scales: 0.25
        scales: 0.5
        scales: 1.0
        scales: 2.0
        aspect_ratios: 0.5
        aspect_ratios: 1.0
        aspect_ratios: 2.0
      }
    }
    first_stage_box_predictor_conv_hyperparams {
      op: CONV
      regularizer {
        l2_regularizer {
          weight: 0.0
        }
      }
      initializer {
        truncated_normal_initializer {
          stddev: 0.009999999776482582
        }
      }
    }
    first_stage_nms_score_threshold: 0.0
    first_stage_nms_iou_threshold: 0.699999988079071
    first_stage_max_proposals: 100
    first_stage_localization_loss_weight: 2.0
    first_stage_objectness_loss_weight: 1.0
    initial_crop_size: 14
    maxpool_kernel_size: 2
    maxpool_stride: 2
    second_stage_box_predictor {
      mask_rcnn_box_predictor {
        fc_hyperparams {
          op: FC
          regularizer {
            l2_regularizer {
              weight: 0.0
            }
          }
          initializer {
            variance_scaling_initializer {
              factor: 1.0
              uniform: true
              mode: FAN_AVG
            }
          }
        }
        use_dropout: false
        dropout_keep_probability: 1.0
      }
    }
    second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.0
        iou_threshold: 0.6000000238418579
        max_detections_per_class: 100
        max_total_detections: 100
      }
      score_converter: SOFTMAX
    }
    second_stage_localization_loss_weight: 2.0
    second_stage_classification_loss_weight: 1.0
  }
}
train_config {
  batch_size: 2
  data_augmentation_options {
    normalize_image {
            original_minval = 0,
            original_maxval = 255,
            target_minval = 0,
            target_maxval = 1
    }
  }
  data_augmentation_options {
    normalize_image {
            original_minval = 0,
            original_maxval = 1,
            target_minval = 0,
            target_maxval = 255
    }
  }
  optimizer {
    momentum_optimizer {
      learning_rate {
        constant_learning_rate {
          learning_rate: 0.0015249886782839894
        }
      }
      momentum_optimizer_value: 0.8637624382972717
    }
    use_moving_average: false
    moving_average_decay: 0.9997233748435974
  }
  gradient_clipping_by_norm: 10.0
  fine_tune_checkpoint: "weights/model.ckpt"
  from_detection_checkpoint: true
  num_steps: 40
}
train_input_reader {
  label_map_path: "data/DETAILED_label_map.pbtxt"
  queue_capacity: 110
  min_after_dequeue: 100
  num_readers: 8
  tf_record_input_reader {
    input_path: "data/DETAILED_TRAINING_unbalanced.record"
  }
}
eval_config {
  num_visualizations: 0
  num_examples: 100
  max_evals: 1
}
eval_input_reader {
  label_map_path: "data/DETAILED_label_map.pbtxt"
  shuffle: false
  queue_capacity: 1
  min_after_dequeue: 0
  num_epochs: 1
  num_readers: 1
  tf_record_input_reader {
    input_path: "data/DETAILED_VALIDATION.record"
  }
}
```

**Comment by lysukhin** [comment]
Probably the same issue as https://github.com/tensorflow/models/issues/2066

**Comment by lysukhin** [comment]
By the way, in your `normalize_image` block you should use `:` instead of `=` in order to make it work at all.

**Comment by tetchart3** [comment]
Thank you! And yes, I have been using ```:``` instead of ```=``` in my actual configs. I had a typo when I wrote the question.

**Comment by Marjanmoodi** [comment]
@tyleretchart Could you fix the problem? How we should normalize images? Does the API do it automatically?
